pkgs/by-name/ta/taler-exchange/package.nix

```diff
@@ -90,6 +104,52 @@ stdenv.mkDerivation (finalAttrs: {
     popd
   '';
 
+  postFixup = ''
+    # Wrap scripts with necessary runtime dependencies
+    for file in \
+      taler-config \
+      taler-exchange-helper-measure-test-{form,oauth} \
+      taler-exchange-helper-converter-oauth2-test-full_name \
+      taler-exchange-kyc-aml-pep-trigger.sh \
+      taler-exchange-kyc-kycaid-converter.sh \
+      taler-exchange-kyc-oauth2-{challenger,nda,test-converter}.sh \
+      taler-exchange-kyc-persona-converter.sh \
+      ; do
+        wrapProgram $out/bin/$file \
+          --prefix PATH : ${
+            lib.makeBinPath [
+              gnunet
+              jq
+              wget
+            ]
+          }
+    done
+
+    # Put scripts with big dependency closures in separate outputs
+    mkdir -p {$auditor,$terms}/bin
+
+    mv $out/bin/taler-terms-generator $terms/bin/taler-terms-generator
+    wrapProgram $terms/bin/taler-terms-generator \
+      --prefix PATH : ${
+        lib.makeBinPath [
+          ghostscript_headless
+          gnumake
+          groff
+          pandoc
+          sphinx
```

Comment:
needs `taler-config`, so we might have to make this into a separate package (or drop it outright).

Reply:
Thanks, I was testing this in a non-pure environment so I didn't notice. Adding the `taler-exchange` derivation output to the `PATH` makes `taler-config` available to the command, which I think is no different from adding `taler-exchange` to the `buildInputs` if we were to move `taler-terms-generator` into a separate derivation.